lib/elixir/src/elixir_expand.erl

```diff
@@ -8,7 +8,13 @@
 expand({'=', Meta, [Left, Right]}, E) ->
   assert_no_guard_scope(Meta, "=", E),
   {ERight, ER} = expand(Right, E),
-  {ELeft, EL}  = elixir_clauses:match(fun expand/2, Left, E),
+  {ELeft, EL} = elixir_clauses:match(fun expand/2, Left, E),
+  if
+    element(1, ELeft) == '<<>>' ->
+      refute_parallel_bitstring_match(ERight, E, ?key(E, context) == match);
```

Comment:
Wouldn't it be better to pass the context downstream instead of true/false?

Reply:
Actually, nevermind. :)

Reply:
Btw, ELeft can be a list, it is not always guaranteed to be a tuple. And we could also have entries like this:

    {:<<>>, ...} = ...

So I think we need to pattern match on a tuple with three elements.